6_nodejs/week1/avg.js

```diff
@@ -0,0 +1,23 @@
+const args = process.argv.slice(2);
+
+if(args.length === 0){
+    console.log("No arguments provided");
+}else{
+    let sum = 0;
+    let count = 0;
```

Comment:
It might be a bit overkill to handle a count, instead of just using args.length, but with your isNaN check, i see how it makes sense. You could also use `args.filter((arg) => !isNan(arg)).length`